cartridges/int_cloudinary/cartridge/scripts/jobs/steps/cldUploadContentLibraryAssets.js

```diff
@@ -130,10 +131,14 @@ module.exports.Start = function (args) {
             }
         }
 
+        var runTime = new Date();
         Transaction.wrap(function () {
-            var runTime = new Date();
             currentSite.preferences.custom.CLDContentLibraryJobLastExecutionDate = runTime;
         });
+        var CLDContentLibraryJobLastExecutionDate = currentSite.preferences.custom.CLDContentLibraryJobLastExecutionDate ? currentSite.preferences.custom.CLDContentLibraryJobLastExecutionDate.toString() : currentSite.preferences.custom.CLDContentLibraryJobLastExecutionDate;
+        if (runTime.toString() !== CLDContentLibraryJobLastExecutionDate) {
+            jobLogger.warn('Unable to update date, date should be : {0}', runTime.toString());
```

Comment:
Please update the message to "Unable to update Job Last Execution Date in Custom Preferences..., date should be..."

Reply:
@asad-rafter , I should have been clearer, apologizes. The purpose of the message is to let the user know where the issue is.
"Unable to update the job last execution timestamp in Custom Preferences->Cloudinary Jobs Configurations field: Content Library Job Last Execution Date"

Also, please add this validation to the other delta updates:
Update Product Feeds Job Last Execution Date
Catalog Content Job Last Execution Date